internal/docker/handler_channel.go

```diff
@@ -253,10 +256,50 @@ func (c *channelHandler) OnWindow(_ uint64, columns uint32, rows uint32, _ uint3
 	return c.exec.resize(ctx, uint(rows), uint(columns))
 }
 
+func (c *channelHandler) OnX11Request(
+	requestID uint64,
+	singleConnection bool,
+	proto string,
+	cookie string,
+	screen uint32,
+	reverseHandler sshserver.ReverseForward,
+) error {
+	if c.x11 {
+		return fmt.Errorf("X11 forwarding already setup for this channel")
+	}
+
+	c.networkHandler.logger.Debug("Setting X11 vars")
```

Comment:
These debug messages need to be cleaned up so they are user-compatible. Ideally, we'd log our standardized message/error types.